<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Felipe Melo &amp; José Roberto Botelho" />


<title>Plano de ensino de Ecologia Numérica</title>

<script src="site_libs/header-attrs-2.6/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>





<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 60px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h2 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h3 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h4 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h5 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h6 {
  padding-top: 65px;
  margin-top: -65px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Ecologia Numérica</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Início</a>
</li>
<li>
  <a href="exemplo.html">Exemplo</a>
</li>
<li>
  <a href="Links.html">Links</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Aulas
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="plano_ensino.html">Ementa</a>
    </li>
    <li>
      <a href="base_dados.html">Aula 1</a>
    </li>
    <li>
      <a href="med_assoc.html">Aula 2</a>
    </li>
    <li class="divider"></li>
    <li>
      <a href="page-c.html">Aula 3</a>
    </li>
    <li>
      <a href="page-d.html">Aula 4</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Plano de ensino de <em>Ecologia Numérica</em></h1>
<h4 class="author">Felipe Melo &amp; José Roberto Botelho</h4>
<h4 class="date">25/01/2021</h4>

</div>


<div id="carga-horária" class="section level2">
<h2>Carga horária</h2>
<p>45 horas</p>
</div>
<div id="ementa" class="section level2">
<h2>Ementa</h2>
<p>Ecologia numérica é uma disciplina que trata da análise da biodiversidade em comunidades biológicas através de técnicas de análise multivariada e índices de diversidade taxonômica, funcional e filogenética</p>
</div>
<div id="objetivos" class="section level2">
<h2>Objetivos</h2>
<p>O objetivo da disciplina é fornecer ferramentas analíticas sobre a biodiversidade através de métricas diversas de diversidade biológica em seus distintos níveis: taxonômica, funcional e filogenética. Ainda, oferecer ao estudante os conceitos básicos de análises de comunidades biológicas através e métodos multivariados utilizados na ecologia de comunidades.</p>
</div>
<div id="metodologia" class="section level2">
<h2>Metodologia</h2>
<p>A disciplina será ministrada através de aulas expositivas e exercícios práticos com comunidades naturais e simuladas e com a utilização do programa R como principal software de análise de dados</p>
</div>
<div id="avaliação" class="section level2">
<h2>Avaliação</h2>
<p>A avaliação será através de provas escritas e seminários em grupo com os estudantes e exercícios periódicos para a fixação dos conteúdos</p>
</div>
<div id="conteúdo-programático" class="section level2">
<h2>Conteúdo programático</h2>
<ul>
<li>Conceitos básicos de diversidade biológica</li>
<li>Diversidade taxonômica: índices de diversidade biológica; diversidade verdadeira (números de Hill)</li>
<li>Exercícios de descrição de comunidades com índices de diversidade</li>
<li>Métodos de ordenação de comunidades biológicas (NMDS; PCA; RDA; CCA)</li>
<li>Estimadores de diversidade biológica (estimadores não-paramétricos; rarefação; interpolação)</li>
<li>Diversidade Funcional</li>
<li>Exercícios de descrição funcional de comunidades biológicas</li>
<li>Métodos de cálculo da diversidade funcional (Dispersão Funcional, Médias ponderadas de comunidades)</li>
<li>Diversidade filogenética</li>
<li>Exercícios de descrição filogenética de comunidades biológicas</li>
<li>Métodos de diversidade filogenética (MNTD, NRI, MPD)</li>
<li>Aplicações da ecologia numérica nos estudos de diversidade biológica.</li>
<li>Análise distribucional: curvas de dominância</li>
</ul>
</div>
<div id="bibliografia" class="section level2">
<h2>Bibliografia</h2>
<div id="programação-em-r" class="section level3">
<h3>Programação em R</h3>
<p>Adler, J. (2012). R in a nutshell: A desktop quick reference. O’Reilly Media, Inc.</p>
<p>Allesina, S., &amp; Wilmes, M. (2019). Computing Skills for Biologists: A Toolbox. Princeton University Press.</p>
<p>Beckerman, A. P., Childs, D. Z., &amp; Petchey, O. L. (2017). Getting started with R: an introduction for biologists. Oxford University Press.</p>
<p>Braun, W. J., &amp; Murdoch, D. J. (2016). A first course in statistical programming with R. Cambridge University Press.</p>
<p>Burns, P. (2011). The R inferno. Lulu.com.</p>
<p>Chambers, J. (2008). Software for data analysis: programming with R. Springer Science &amp; Business Media.</p>
<p>Chambers, J. M. (2017). Extending R. Chapman and Hall/CRC.</p>
<p>Campbell, M. (2019). Learn RStudio IDE: Quick, Effective, and Productive Data Science. Springer.</p>
<p>Chang, W. (2018). R graphics cookbook: practical recipes for visualizing data. O’Reilly Media.</p>
<p>Cotton, R. (2013). Learning R: A Step-by-Step Function Guide to Data Analysis. O’Reilly Media, Inc.</p>
<p>Cotton, R. (2017). Testing R Code. Chapman and Hall/CRC.</p>
<p>Crawley, M. J. (2012). The R book. John Wiley &amp; Sons.</p>
<p>Davies, T. M. (2016). The Book of R: A first course in programming and statistics. No Starch Press.</p>
<p>Engel C. 2019. Introduction to R. <a href="https://cengel.github.io/R-intro/">https://cengel.github.io/R-intro/</a></p>
<p>Field, A., Miles, J., &amp; Field, Z. (2012). Discovering statistics using R. Sage publications.</p>
<p>Gandrud, C. (2016). Reproducible research with R and R studio. Chapman and Hall/CRC.</p>
<p>Gardener, M. (2012). The essential R reference. John Wiley &amp; Sons.</p>
<p>Gentleman, R. (2008). R programming for bioinformatics. Chapman and Hall/CRC.</p>
<p>Gillespie, C., &amp; Lovelace, R. (2016). Efficient R programming: a practical guide to smarter programming. O’Reilly Media, Inc.</p>
<p>Grolemund, G. (2014). Hands-On Programming with R: Write Your Own Functions and Simulations. O’Reilly Media, Inc.</p>
<p>Hastie, T., Tibshirani, R., &amp; Friedman, J. (2009). The elements of statistical learning: data mining, inference, and prediction. Springer Science &amp; Business Media. <a href="https://web.stanford.edu/~hastie/ElemStatLearn/">https://web.stanford.edu/~hastie/ElemStatLearn/</a></p>
<p>Healy, K. (2018). Data visualization: a practical introduction. Princeton University Press.</p>
<p>Hector, A. (2015). The new statistics with R: an introduction for biologists. Oxford University Press.</p>
<p>Hilfiger, J. J. (2015). Graphing Data with R: An Introduction. O’Reilly Media, Inc.</p>
<p>Holmes, S., &amp; Huber, W. (2018). Modern statistics for modern biology. Cambridge University Press. <a href="http://web.stanford.edu/class/bios221/book/index.html">http://web.stanford.edu/class/bios221/book/index.html</a></p>
<p>Irizarry, R. A., &amp; Love, M. I. (2016). Data Analysis for the Life Sciences with R. Chapman and Hall/CRC. <a href="http://www.rwdc2.com/files/rafa.pdf">http://www.rwdc2.com/files/rafa.pdf</a></p>
<p>Ismay, C., &amp; Kim, A. Y. (2019). Statistical Inference via Data Science: A ModernDive into R and the Tidyverse. CRC Press. <a href="https://moderndive.com/">https://moderndive.com/</a></p>
<p>James, G., Witten, D., Hastie, T., &amp; Tibshirani, R. (2013). An introduction to statistical learning (Vol. 112, p. 18). New York: springer. <a href="http://faculty.marshall.usc.edu/gareth-james/ISL/">http://faculty.marshall.usc.edu/gareth-james/ISL/</a></p>
<p>Jones, O., Maillardet, R., &amp; Robinson, A. (2014). Introduction to scientific programming and simulation using R. Chapman and Hall/CRC.</p>
<p>Kabacoff, R. (2015). R in Action: Data Analysis and Graphics With R. Greenwich, CT.</p>
<p>Lander, J. P. (2017). R for everyone: advanced analytics and graphics. Pearson Education.</p>
<p>Maindonald, J., &amp; Braun, J. (2010). Data analysis and graphics using R: an example-based approach. Cambridge University Press.</p>
<p>Matloff, N. (2011). The art of R programming: A tour of statistical software design. No Starch Press.</p>
<p>McGrath, M. (2018). R for Data Analysis in easy steps - R Programming essentials. In Easy Steps Limited.</p>
<p>Micheaux, P. L., Drouilhet, R., &amp; Liquet, B. (2013). The R software The R Software: Fundamentals of Programming and Statistical Analysis. Springer Science &amp; Business Media.</p>
<p>Oliveira, P. F., Guerra, S., Mcdonnell, R. (2018). Ciência de dados com R – Introdução. IBPAD. <a href="https://cdr.ibpad.com.br/">https://cdr.ibpad.com.br/</a></p>
<p>R Core Team. (2019). R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. <a href="https://www.r-project.org/">https://www.r-project.org/</a></p>
<p>Racine, J. S. (2012). RStudio: a platform‐independent IDE for R and Sweave. Journal of Applied Econometrics, 27(1), 167-172.</p>
<p>Shahbaba, B. (2012). Biostatistics with R: An Introduction to Statistics Through Biological Data. Springer</p>
<p>Silge, J., &amp; Robinson, D. (2017). Text mining with R: A tidy approach. O’Reilly Media, Inc.</p>
<p>Teetor, P. (2019). R cookbook: proven recipes for data analysis, statistics, and graphics. O’Reilly Media, Inc.</p>
<p>Wickham, H. (2015). R packages: organize, test, document, and share your code. O’Reilly Media, Inc.</p>
<p>Wickham, H. (2016). ggplot2: elegant graphics for data analysis. Springer.</p>
<p>Wickham, H. (2019). Advanced r. Chapman and Hall/CRC.</p>
<p>Wickham, H., &amp; Grolemund, G. (2016). R for data science: import, tidy, transform, visualize, and model data. O’Reilly Media, Inc. <a href="https://r4ds.had.co.nz/">https://r4ds.had.co.nz/</a></p>
<p>Wiley, M., &amp; Wiley, J. F. (2016). Advanced R Data Programming and the Cloud. Apress.</p>
<p>Wilke, C. O. (2019). Fundamentals of Data Visualization: A Primer on Making Informative and Compelling Figures. O’Reilly Media. <a href="https://serialmentor.com/dataviz/">https://serialmentor.com/dataviz/</a></p>
<p>Xie, Y., Allaire, J. J., &amp; Grolemund, G. (2018). R markdown: The definitive guide. CRC Press. <a href="https://bookdown.org/yihui/rmarkdown/">https://bookdown.org/yihui/rmarkdown/</a></p>
<p>Zumel, N., &amp; Mount, J. (2014). Practical data science with R. Manning Publications Co.</p>
<p>Zuur, A., Ieno, E. N., &amp; Meesters, E. (2009). A Beginner’s Guide to R. Springer Science &amp; Business Media.</p>
</div>
<div id="conteúdo-teórico" class="section level3">
<h3>Conteúdo teórico</h3>
<p>MAGURRAN, A. E. Ecological diversity and its measurement. University press, Cambridge, 1988. 167p.</p>
<p>Chao, A., Chiu, C.-H. &amp; Jost, L. (2014) Unifying Species Diversity, Phylogenetic Diversity, Functional Diversity, and Related Similarity and Differentiation Measures Through Hill Numbers. Annual Review of Ecology, Evolution, and Systematics, 45, 297–324.</p>
<p>Cavender-Bares, J., Kozak, K.H., Fine, P.V.A. &amp; Kembel, S.W. (2009) The merging of community ecology and phylogenetic biology. Ecology Letters, 12, 693–715.</p>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
